utest/running/test_running.py

```diff
@@ -251,5 +252,16 @@ def test_reset_logging_conf(self):
         assert_equals(logging.raiseExceptions, 1)
 
 
+class TestListeners(RunningTestCase):
+
+    def test_listeners(self):
+        module_file = join(ROOT, 'utest', 'resources', 'Listener.py')
+        suite = build('setups_and_teardowns.robot')
+        suite.run(output=None, log=None, report=None, listener=module_file+":1")
+        self._assert_outputs([("[from listener 1]", 1)])
+        suite.run(output=None, log=None, report=None)
+        self._assert_outputs([("[from listener 1]", 1)])
```

Comment:
Do I get it right that the verified `"[from listener 1]"` was already created by the first run and here it is tested that the second run didn't produce new output? I think it would be better to add a method like `RunningTestCase._clear_outputs` that could be called before the second run and then verified that outputs are empty after the second run.